### ISSUE TITLE
token?.hasActionContainer is not a function

### ISSUE THREAD

**Issue by Djahal** [trioderegion/dnd5e-helpers#191]
**Foundry VTT** : 0.8.8

**Dnd5e** : 1.4.1

**DnD5e Helpers** : 3.0.2

When I end a combat via the combat tracker I get this error : 

![image](https://user-images.githubusercontent.com/80932931/134065434-d3233b8d-b46a-4f16-9ae8-262430bcc8a0.png)

Do not hesitate if you need any further information :).

**Comment by trioderegion** [comment]
ahh, I think this is similar to an issue I just fixed, but in the opposite order.

Is it a critical issue -- i.e. is it breaking your game?

**Comment by Djahal** [comment]
No, currently it's not breaking my game, the combat just ends but I noticed this error as I always prep with the console open.
